src/posts/blogs/Blog.style.js

```diff
@@ -43,7 +32,7 @@ export const BlogWrapper = styled.div`
       border: 2px solid gray;
     }
     .text-centre {
```

Comment:
I wonder if we might move away from this British spelling of "center" at some point.